### ISSUE TITLE
Address webaim accessibility issues

### ISSUE THREAD

**Issue by vincerubinetti** [greenelab/lab-website-template#101]
Example:
https://wave.webaim.org/report#/https://www.waysciencelab.com/

Some things to correct:

- heading anchors should have an `aria-label` like `Perma-link to [this section]`
- presentational images in components like feature should have `role="presentation"` so screen readers know they can be skipped. in general, need to take a close look at all places where images can be and either bypass, or allow website maintainer to reliably add alt text. i know i've done this in a few places, but not all/enough.
- get rid of all font sizes < `1.0rem` and just round them up to 1 to hopefully avoid complaints about small font sizes from GSC
- add lang attr to html
- maybe make footer icons bigger for mobile touch accessibility

Run through Lighthouse for more.
